src/store/transaction/transaction.reducer.js

```diff
@@ -234,7 +234,7 @@ function transactionReducer (state = initialTransactionState, action) {
             ...state.steps[STEP_NAME.REVIEW_TRANSACTION],
             transaction: {
               exit: action.exit,
-              amount: action.exit.balance,
+              amount: action.exit.balance || action.exit.amount,
```

Comment:
Why an exit can have `balance` and `amount`?

Reply:
Balance is when we use the Exit object returned by the API, amount when it comes from the storage 😔